src/endpoints/pool/entities/transaction.in.pool.dto.ts

```diff
@@ -0,0 +1,50 @@
+import { Field, ObjectType } from "@nestjs/graphql";
+import { ApiProperty } from "@nestjs/swagger";
+import { TransactionType } from "src/endpoints/transactions/entities/transaction.type";
+
+@ObjectType("TransactionInPool", { description: "Transaction in pool object type." })
+export class TransactionInPool {
+  constructor(init?: Partial<TransactionInPool>) {
+    Object.assign(this, init);
+  }
+
+  @Field(() => String, { description: "Transaction hash." })
+  @ApiProperty({ type: String, example: "" })
```

Comment:
To have a better exposure of all fields, will be greate to add some examples for every fields

Reply:
done